src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -29,36 +29,24 @@ class LogicalMeter:
         creating `LogicalMeter` instances.
 
     Example:
-        ```python
+        import asyncio
         from datetime import timedelta
 
         from frequenz.sdk import microgrid
-        from frequenz.sdk.timeseries import ResamplerConfig
+        from frequenz.sdk.actor import ResamplerConfig
+        from frequenz.client.microgrid import ComponentMetricId
 
         await microgrid.initialize(
-            "grpc://127.0.0.1:50051",
-            ResamplerConfig(resampling_period=timedelta(seconds=1))
+            "microgrid.sandbox.api.frequenz.io",
+            62060,
```

Comment:
Now `initialize()` takes a connection URL instead.
```suggestion
            "grpc://microgrid.sandbox.api.frequenz.io:62060",
```